src/lib/notifications/config.js

```diff
@@ -0,0 +1,24 @@
+import { appEnv } from '../utils/env'
+
+const DefaultNotificationConfig = {
+  notificationSchedule: 'minute', // repeat in each minute
+  notificationTime: new Date(Date.now() + 60 * 1000), // 1 minute after app been started
+}
+
+const NotificationProdConfig = {
+  notificationSchedule: 'day', // repeat daily
+  notificationTime: (() => {
+    // 12 PM UTC
+    const date = new Date()
+
+    date.setUTCHours(12, 0, 0, 0)
+    return date
+  })(),
+}
+
+const NotificationConfig =
+  {
+    production: NotificationProdConfig,
+  }[appEnv] || DefaultNotificationConfig
```

Comment:
do not need to place it here, move to config.js
also, it should be determined not by the env name but by some env flag-variable (like test mode is configured currently)